README.md

```diff
@@ -279,6 +279,24 @@ role.
 Similar to `controller` and `vlan`, the `parent` references the connection profile in
 the ansible role.
 
+#### `type: infiniband`
```

Comment:
Mention, that this is only supported for the NM provider.

Reply:
Updated.